api/controllers/roomController.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import roomService from "../services/roomService";
+
+class roomController {
+  static getRoomList = async (req: Request, res: Response) => {
+    const result = await roomService.getRoomList(req);
```

Comment:
```suggestion
    const result = await roomService.getRoomList();
```
보니까 Dao까지 req 인수를 보낼 이유가 없는 것 같습니다.